contents/scummvm.oscmeta

```diff
@@ -0,0 +1,23 @@
+{
+    "information": {
+        "name": "ScummVM",
+        "author": "The ScummVM Team",
+        "category": "emulators",
+        "peripherals": ["Wii Remote", "GameCube Controller", "SDHC"],
+        "version": "2.7.0"
+    },
+    "source": {
+        "type": "manual"
+    },
+	"treatments": [
+		{
+			"treatment": "web.download", "arguments": ["https://downloads.scummvm.org/frs/scummvm/2.7.0/scummvm-2.7.0-wii.zip", "/"]
```

Comment:
invalid save location..

Reply:
that means i need to save them into a specific folder, extract them into another folder and then delete the folder with the archive? is that a valid option?

Reply:
I don't understand your question.
you need to be downloading the zip to unimportant location X, extract it to location Y, and then delete the no longer needed zip at location X